### PR TITLE
Filtered out retired and draft tutor books

### PR DESCRIPTION
**Pull request by edwoodward**

### CODE REVIEW

**Review by mwvolo — APPROVED** [review]

File: pages/models.py [hunk 1]
```diff
@@ -2179,12 +2179,13 @@ def tutor_books(self):
         books = Book.objects.filter(tutor_marketing_book=True).order_by('path')
         book_data = []
         for book in books:
-            book_data.append({
-                'id': book.id,
-                'slug': 'books/{}'.format(book.slug),
-                'title': book.title,
-                'cover_url': book.cover_url,
-            })
+            if book.book_state is not 'Retired' and book.book_state is not 'Draft':
```
Comment (mwvolo) [inline pages/models.py:2182]: I think you can shorten this to book.book_state not in ['Retired', 'Draft']

Comment (edwoodward) [reply pages/models.py]: Done
